numpy/lib/tests/test_arraysetops.py

```diff
@@ -346,74 +346,53 @@ def test_in1d_invert(self, kind):
                             dtype=np.float32)
                 b = [2, 3, 4] * mult
                 b = np.array(b, dtype=np.float32)
-                assert_array_equal(np.invert(in1d(a, b, kind=kind)),
-                                   in1d(a, b, invert=True, kind=kind))
+                assert_array_equal(np.invert(isin(a, b, kind=kind)),
+                                   isin(a, b, invert=True, kind=kind))
 
-    @pytest.mark.parametrize("kind", [None, "sort", "table"])
-    def test_in1d_ravel(self, kind):
-        # Test that in1d ravels its input arrays. This is not documented
-        # behavior however. The test is to ensure consistentency.
-        a = np.arange(6).reshape(2, 3)
-        b = np.arange(3, 9).reshape(3, 2)
-        long_b = np.arange(3, 63).reshape(30, 2)
-        ec = np.array([False, False, False, True, True, True])
-
-        assert_array_equal(in1d(a, b, assume_unique=True, kind=kind),
-                           ec)
-        assert_array_equal(in1d(a, b, assume_unique=False,
-                                kind=kind),
-                           ec)
-        assert_array_equal(in1d(a, long_b, assume_unique=True,
-                                kind=kind),
-                           ec)
-        assert_array_equal(in1d(a, long_b, assume_unique=False,
-                                kind=kind),
-                           ec)
```

Comment:
It would be great if another reviewer could confirm it's OK to delete this. I wonder if someone is depending on this behavior, maybe it needs a note somewhere in a NumPy 2.0 API migration guide?

Reply:
So the difference is that `np.in1d(a, b)` squashes output to one dimension, where `np.isin(a, b)` retains `a`'s shape for the output. 

I think `np.isin(a, b).ravel()` is enough to get same behavior as deprecated `in1d`, which I can note in the migration guide.